addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -78,11 +78,24 @@
 
                 display: table-cell;
                 @include text-truncate();
-                vertical-align: middle;
 
                 &.o_list_text {
                     white-space: pre-wrap;
                 }
+
+                // TODO ugly temporary overrides to visually align the field widget
+                // icons in the list view without the vertical-align: middle;
+                .o_field_priority, .o_field_list_activity .fa {
+                    margin-top: .1em;
+                }
+
+                .o_field_project_task_state_selection {
+                    margin-top: .2em;
+                }
+
+                .o_field_many2one_avatar_resource .fa-wrench {
+                    margin-top: .25em
+                }
```

Comment:
Beside `o_field_priority`, all the other classes are from specific modules (ie. mail, project, planning) ; this shouldn't be in `/web`.

Although I see your goal to regroup them in the same place to easily remove the "ugliness" at once later on... Those are way too specific and I suggest that you place them in their related modules, with the same "FIXME" for each.